### PR TITLE
 Add classes to read the file kitodo_fileFormats.xml 

### PR DESCRIPTION
**Pull request by matthias-ronge**
The `kitodo_fileFormats.xml` file shall be used to configure different file formats with a label and settings that belong together, such as MIME type and file extension. It can then be used to configure variaous correspondences in Kitodo that belong to it, such as Java IO and Validation.

### CODE REVIEW

File: Kitodo/src/test/java/org/kitodo/config/xml/fileformats/FileFormatsConfigIT.java [hunk 1]
```diff
@@ -0,0 +1,46 @@
+package org.kitodo.config.xml.fileformats;
```
Comment (henning-gerhardt) [inline Kitodo/src/test/java/org/kitodo/config/xml/fileformats/FileFormatsConfigIT.java]: Missing license header.

File: Kitodo/src/main/java/org/kitodo/config/xml/fileformats/FileFormatsConfig.java [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+     *
+     * @return the configured file formats
+     * @throws IOException
+     *             if reading from the file fails
+     * @throws JAXBException
+     *             if the content of the file is syntactically or semantically
+     *             incorrect
+     */
+    public static List<FileFormat> getFileFormats() throws IOException, JAXBException {
+        KitodoServiceLoader<FileManagementInterface> fileManagementInterface = new KitodoServiceLoader<>(
```
Comment (henning-gerhardt) [inline Kitodo/src/main/java/org/kitodo/config/xml/fileformats/FileFormatsConfig.java]: Accessing local application configuration files should not be done over the file management module as this is only for accessing meta data and pictures which could be delivered from a remote repository in an other implementation.

File: Kitodo/src/test/java/org/kitodo/config/xml/fileformats/FileFormatsConfigIT.java [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+import static org.hamcrest.MatcherAssert.assertThat;
```
Comment (henning-gerhardt) [inline Kitodo/src/test/java/org/kitodo/config/xml/fileformats/FileFormatsConfigIT.java]: Dependency of `org.hamcrest` is not defined in core module of Kitodo. If you want to use it, add this library to the `Kitodo/pom.xml` for scope `test`.

**Comment by henning-gerhardt** [comment]
Please rebase against master as #1669 is now merged and without this changes Travis CI will fail every time.

File: Kitodo/src/test/java/org/kitodo/config/xml/fileformats/FileFormatsConfigIT.java (hunk 1 above)
Comment (matthias-ronge) [reply Kitodo/src/test/java/org/kitodo/config/xml/fileformats/FileFormatsConfigIT.java]: Fixed

File: Kitodo/src/test/java/org/kitodo/config/xml/fileformats/FileFormatsConfigIT.java (hunk 3 above)
Comment (matthias-ronge) [reply Kitodo/src/test/java/org/kitodo/config/xml/fileformats/FileFormatsConfigIT.java]: Fixed

**Comment by matthias-ronge** [comment]
Rebase is not possible as this branch is already merged into one of my other branches because I need it there as prerequisite, and I cannot merge the latest changes again if I rebase it. So I had to do a merge here instead.

**Pull request merged**